vcd/resource_vcd_vapp_vm.go

```diff
@@ -1789,6 +1976,22 @@ func resourceVcdVAppVmUpdateExecute(d *schema.ResourceData, meta interface{}, ex
 			if err != nil {
 				return diag.Errorf(errorCompletingTask, err)
 			}
+
+			err = vm.Refresh()
+			if err != nil {
+				return diag.Errorf("error refreshing VM: %s", err)
+			}
+
+			if enterBiosSetup := d.Get("boot_options.0.enter_bios_setup").(bool); enterBiosSetup {
+				biosSetup := &types.BootOptions{
+					EnterBiosSetup: &enterBiosSetup}
+
+				vm, err = vm.UpdateBootOptions(biosSetup)
+				if err != nil {
+					return diag.Errorf("error enabling BIOS setup: %s", err)
+				}
+			}
+
```

Comment:
Will it not ignore entering bios if `customizationNeeded=true`?

Reply:
Yes, I missed this part, updated.